### PR TITLE
Make project more approachable by other developers

### PR DESCRIPTION
**Pull request by SarenCurrie**
Maybe I can even get the project to the point that it's actually useful to people outside our own discord server, who knows?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
 # discord-r9k
+
+An experimental [Robot 9000](https://blog.xkcd.com/2008/01/14/robot9000-and-xkcd-signal-attacking-noise-in-chat/) for Discord.
+
+## Usage
+
+First, o to the [Discord developer documentation](https://discordapp.com/developers/), create an app and get your token.
```
Comment (sadflask) [inline README.md]: "go to the"

File: README.md [hunk 2]
```diff
@@ -1 +1,56 @@
...
+
+To add the bot to your server, insert the client ID into this URL https://discordapp.com/oauth2/authorize?client_id=CLIENT_ID_GOES_HERE&scope=bot&permissions=0 and paste it into your browser of choice.
+
+The bot as it is currently has a lot of things tuned to my personal discord server, which I plan to refactor away at some point, in the mean time, you will probably need to head into the code and take these out if they end up annoying you.
+
+## Documentation
+
+### `addMessage(trigger, callback)`
+
+#### `tirgger`
```
Comment (sadflask) [inline README.md]: "trigger" (ed :D)

File: README.md [hunk 3]
```diff
@@ -1 +1,56 @@
...
+
+A String, RegExp or Array, that will trigger this event.
```
Comment (harveyrendell) [inline README.md]: I would change this to list the valid types e.g.

The trigger can be one of the following types:
* **String** - The callback will be triggered if the message matches the String exactly
* **Regular Expression** - if the message matches the RegExp, the callback will be triggered, the callback argument will include the list of matches returned by [`String.match()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match)
* **Array** -  All array elements are treated as separate triggers.

Comment (harveyrendell) [reply README.md]: Also `s/seperate/separate/`

File: README.md [hunk 4]
```diff
@@ -1 +1,56 @@
 # discord-r9k
+
+An experimental [Robot 9000](https://blog.xkcd.com/2008/01/14/robot9000-and-xkcd-signal-attacking-noise-in-chat/) for Discord.
```
Comment (harveyrendell) [inline README.md]: Maybe add a sentence or two describing the other things the bot currently does/supports.

**Review by harveyrendell — CHANGES_REQUESTED** [review]
Would be nice to have a CHANGELOG to go with the bot so we know what version of the code is being run. I might raise something like this as a PR.

**Comment by SarenCurrie** [comment]
Raise a issue for the changelog, I'll address everything else.

**Review by harveyrendell — APPROVED** [review]

**Pull request merged**